src/containers/tab-sidemenu/todos-menu.tsx

```
@@ -0,0 +1,21 @@
+import DashBoard from "@/assets/dashboard.svg";
+
+import CustomButton from "./custom-button";
+import Menus from "./menus";
```

ESLint

[error] Missing file extension for "./menus"

(import/extensions)